addons/mail/static/src/core/common/message_preview_prefix.js

```diff
@@ -0,0 +1,12 @@
+import { Component } from "@odoo/owl";
+
+/**
+ * @typedef {Object} Props
+ * @property {import("models").Message} message
+ * @property {import("models").ResPartner|import("models").MailGuest} [persona]
+ * @extends {Component<Props, Env>}
+ */
+export class MessagePreviewPrefix extends Component {
+    static props = ["message", "persona?"];
```

Comment:
Persona is not required, and the name is not explicit. What we want is:
- Am I the author of the message? `fa-reply` + `You:`
- Is is a 1 to 1 chat? show nothing.
- Else, show the author name.

Reply:
A test was failling, I'll try again